webapp/src/main/webapp/news-details/components/ExoNewsArchive.vue

```diff
@@ -8,48 +8,31 @@
         </div>
       </transition>
     </div>
-    <a :data-original-title="archiveLabel" class="btn newsArchiveButton"
```

Comment:
you should display this block only when you call this component from news details, for that you should use an adequate prop for this component